webpack/JobInvocationDetail/JobInvocationOverview.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import DefaultLoaderEmptyState from 'foremanReact/components/HostDetails/DetailsCard/DefaultLoaderEmptyState';
+import { translate as __, documentLocale } from 'foremanReact/common/I18n';
+
+const JobInvocationOverview = ({ data }) => {
+  const {
+    start_at: startAt,
+    ssh_user: sshUser,
+    template_id: templateId,
+    template_name: templateName,
+    effective_user: effectiveUser,
+  } = data;
+
+  const dateOptions = {
+    day: 'numeric',
+    month: 'short',
+    year: 'numeric',
+    hour: '2-digit',
+    minute: '2-digit',
+    hour12: false,
+    timeZoneName: 'short',
+  };
+  const dateConverted = new Date(startAt);
+  const dateLocaleFormatted = dateConverted
+    .toLocaleString(documentLocale(), dateOptions)
+    .replace(/(\d{4}) /, '$1, ');
+  const dateCurrent = new Date();
+
+  return (
+    <DescriptionList
+      columnModifier={{
+        default: '2Col',
+      }}
+      isHorizontal
+      isCompact
+      isFluid
+      isAutoColumnWidths
+    >
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Effective user:')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {effectiveUser || <DefaultLoaderEmptyState />}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Started at:')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {startAt && dateConverted <= dateCurrent
+            ? dateLocaleFormatted
+            : __('Not yet')}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('SSH user:')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {sshUser || <DefaultLoaderEmptyState />}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Template:')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {templateName ? (
+            <Button
```

Comment:
The button should be disabled if the user cant edit the template

Reply:
Thank you, I've just made the updates